### PR TITLE
PMM-7379 Edit alert rule name

### PR DESCRIPTION
**Pull request by Dasio**
[PMM-7379](https://jira.percona.com/browse/PMM-7379)

- [ ] Build: https://github.com/Percona-Lab/pmm-submodules/pull/1549

- [ ] Links to other linked pull requests (optional).
API - https://github.com/percona/pmm/pull/692
---
- [ ] Tests passed.
- [ ] Feature build pass.
- [ ] (Re)requested review.
- [ ] Fix conflicts with target branch.
- [ ] Update API dependency.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

### CODE REVIEW

File: models/rule_helpers.go [hunk 1]
```diff
@@ -192,6 +193,9 @@ func ChangeRule(q *reform.Querier, ruleID string, params *ChangeRuleParams) (*Ru
 	row.Severity = params.Severity
 	row.Filters = params.Filters
 	row.Params = params.RuleParams
+	if params.Summary != "" {
```
Comment (Dasio) [inline models/rule_helpers.go:196]: I'm not sure about this one.
Everything else is set from request.
But If I do it here, it will be breaking change.

Comment (BupycHuk) [reply models/rule_helpers.go:196]: it's interesting case. In all other endpoints we change only fields that we get from request, but doesn't change other ones. And we should keep this behaviour.
IA is still private API, so you can change behaviour.

**Review by BupycHuk — APPROVED** [review]
Can we create a ticket to change behaviour for other fields?

**Comment by Dasio** [comment]
> Can we create a ticket to change behaviour for other fields?

I can. And also what about bool parameters, or to clear fields? I asked Fabio about it, and they are sending values each time. So with this way it is problematic to clear fields. I would suggest try to look on proto wrappers so we can distinguish nil and empty values.
 But then also, what about slice or maps? These ones should be probably required.

**Comment by BupycHuk** [comment]
we add new fields `remove_*` to remove value from the fields which can be removed. API will be used not only by our front-end, but also can be used by custom scripts.

**Comment by BupycHuk** [comment]
BTW please check why Travis CI is faling

**Comment by AlekSi** [comment]
Please check and update/add comments in .proto files – I think they say something about required fields, how absent values are treated, etc

**Comment by Dasio** [comment]
> BTW please check why Travis CI is faling

Tests are failing, but not related to this PR, issue was raised ons lack. Wrong password..

**Review by artemgavrilov — APPROVED** [review]

**Review by oter — APPROVED** [review]

**Pull request merged**